website/templates/includes/checkInModal.html

```
@@ -9,25 +9,27 @@ <h2 id="modalTitle">
-            <div class="form-group">
-                <label for="next_plan_card">What do you plan to do next?</label>
-                <textarea id="next_plan_card" name="next_plan" rows="3" required></textarea>
-            </div>
-            <div class="form-group">
-                <label for="blockers_card">Do you have any blockers?</label>
-                <textarea id="blockers_card" name="blockers" rows="3"></textarea>
-            </div>
-            <button type="submit" class="submit-button">Submit</button>
-        </form>
+        <div class="modal-content bg-white dark:bg-gray-800 text-gray-900 dark:text-gray-200">
+            <form id="checkInForm"
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)